DirectProgramming/C++SYCL_FPGA/Tutorials/Tools/use_library/part2_lib_func_call/src/lib_rtl_spec.xml

```diff
@@ -1,9 +1,9 @@
 <RTL_SPEC>
-  <FUNCTION name="RtlByteswap" module="byteswap_uint">
+  <FUNCTION name="RtlDSPm27x27u" module="dsp_27x27u">
     <ATTRIBUTES>
       <IS_STALL_FREE value="yes"/>
       <IS_FIXED_LATENCY value="yes"/>
-      <EXPECTED_LATENCY value="0"/>
+      <EXPECTED_LATENCY value="2"/>
       <CAPACITY value="0"/>
```

Comment:
how do you know the new latency is 2?

Why is the capacity not also 2? in general, capacity can match latency (the DSP block should be pipelined and can accept a new input every clock cycle, so latency = capacity)

Reply:
the RTL model I used has specified that the latency is 2 for the design.
Let me update the capacity to 2 to match the latency.